src/annotator/guest.coffee

```diff
@@ -192,6 +251,7 @@ module.exports = class Guest extends Delegator
       this.setVisibleHighlights(state)
 
   destroy: ->
+    this._removeElementEvents()
     $('#annotator-dynamic-style').remove()
```

Comment:
This is the only reference to this selector in this module. Is it actually being used for anything?

Reply:
Indeed, this is the only reference in the code. I'll remove it.